sdk/cognitiveservices/cognitiveservices-qnamaker/package.json

```diff
@@ -52,6 +52,5 @@
     "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/cognitiveservices-qnamaker.js.map'\" -o ./dist/cognitiveservices-qnamaker.min.js ./dist/cognitiveservices-qnamaker.js",
     "prepack": "npm install && npm run build"
   },
-  "sideEffects": false,
-  "autoPublish": true
```

Comment:
What is the reason for removing the auto publish feature?

Reply:
Good catch there was a missing flag. I have added it and regenerated